coffeecard/CoffeeCard.WebApi/Controllers/v2/AccountController.cs

```diff
@@ -46,5 +47,24 @@ public async Task<ActionResult> Delete()
             await _accountService.RequestAnonymization(user);
             return NoContent();
         }
+        
+        /// <summary>
+        /// Check if a given email is in use
+        /// </summary>
+        /// <returns>Account information</returns>
+        /// <response code="200">Successful request</response>
+        /// <response code="401">Invalid credentials</response>
+        [HttpPost]
+        [ProducesResponseType(typeof(EmailInUseDTO), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(void), StatusCodes.Status401Unauthorized)]
+        [Route("emailinuse")]
```

Comment:
In Rest, we usually name the action with a verb. This is understood as we _apply_ an action on the REST collection. With that in mind, I would like to find another name than _emailInUse_. 

Suggestions
- `/check-email`
- `/check-duplicate-email`